plugins/browser-plugin-youtube-tracking/src/buildYouTubeEvent.ts

```diff
@@ -73,22 +73,22 @@ function getMediaPlayerEntities(
   eventData?: EventData
 ): MediaEntities {
   const playerStates: Record<string, boolean> = {
-    ended: false,
-    paused: false,
+    [YT.PlayerState.ENDED]: false,
+    [YT.PlayerState.PAUSED]: false,
```

Comment:
This feels a little weird to me as using numbers as keys in an object feels a bit strange (and now wrongly typed). Its an enum (and a number), so should be `Record<number, boolean>`. However , I'm not particularly picky as we used to map the number to a the string in the previous commit which I don't see is any real difference. Maybe just fix the type.

This function is now also quite different to `getYouTubeEntities` above, however it has the same behaviour. It'd be nice to make them consistent.